docs/data-management/preliminary.md

```diff
@@ -65,9 +65,104 @@ When employing high-performance computing (HPC), we provide [some specific guide
             --url https://github.com/{{ secrets.data.gh_repo | default('<organization>/<repo_name>') }}.git \
             --publish-depends ria-storage
     ```
+    
+- [ ] Create a sub-dataset to host the *MRIQC* derivatives. If you will not run *MRIQC* version 23.1.0, replace the version with the one you intend to run in the command below.
```

Comment:
```suggestion
- [ ] Create a sub-dataset to host the *MRIQC* derivatives.
    Remember to set the correct version of the container (in our case {{ settings.versions.mriqc }}).
```

Reply:
Remember to add the config for `{{ settings.versions.mriqc }}` in `/study-settings.yml` so that the replacement happens consistently throughout the SOPs.

Add something like:

```yaml
versions:
  mriqc: '23.1.0'
  fmriprep: 'xxxx'
  dmriprep: 'xxxx'
```